### ISSUE TITLE
scp not fetching anything

### ISSUE THREAD

**Issue by 130s** [130s/hut_10sqft#67]
```
niquita@pudding:~/tmp/scp_test$ ll
total 8.0K
drwxrwxr-x  2 niquita niquita 4.0K Jul 22 17:08 ./
drwxrwxr-x 12 niquita niquita 4.0K Jul 22 17:08 ../
niquita@pudding:~/tmp/scp_test$ scp niquita@130s-serval.local:/home/niquita/data/radio-exercise_1_and_2-xS92XkVKM0Q.webm .
niquita@130s-serval.local's password: 
[rospack] Error: package 'turtlebot_gazebo' not found
[rospack] Error: package 'turtlebot_gazebo' not found
[rospack] Error: package 'turtlebot_stage' not found
[rospack] Error: package 'turtlebot_stage' not found
[rospack] Error: package 'turtlebot_stdr' not found
Agent pid 2344
niquita@pudding:~/tmp/scp_test$ /home/niquita/c_ubuntu/config/bash/rc_130s-serval.bash: line 11: [: missing `]'
ll
total 8.0K
drwxrwxr-x  2 niquita niquita 4.0K Jul 22 17:08 ./
drwxrwxr-x 12 niquita niquita 4.0K Jul 22 17:08 ../
```

And the other way round ends up in the same result.

**Comment by 130s** [comment]
This is still an issue.

**Comment by 130s** [comment]
#188 fixed the error above but `scp` quietly fails to fetch the file.
```
$ scp -v 13-yonbal.local:/home/n13/data/Dropbox/rsj_seminor07.pdf .
Executing: program /usr/bin/ssh host 13-yonbal.local, user (unspecified), command scp -v -f /home/n13/data/Dropbox/rsj_seminor07.pdf
OpenSSH_7.2p2 Ubuntu-4ubuntu2.2, OpenSSL 1.0.2g  1 Mar 2016
debug1: Reading configuration data /home/n13/.ssh/config
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 19: Applying options for *
debug1: Connecting to 13-yonbal.local [192.168.0.20] port 22.
debug1: Connection established.
debug1: identity file /home/n13/.ssh/id_rsa type 1
debug1: key_load_public: No such file or directory
debug1: identity file /home/n13/.ssh/id_rsa-cert type -1
debug1: key_load_public: No such file or directory
debug1: identity file /home/n13/.ssh/id_dsa type -1
debug1: key_load_public: No such file or directory
debug1: identity file /home/n13/.ssh/id_dsa-cert type -1
debug1: key_load_public: No such file or directory
debug1: identity file /home/n13/.ssh/id_ecdsa type -1
debug1: key_load_public: No such file or directory
debug1: identity file /home/n13/.ssh/id_ecdsa-cert type -1
debug1: key_load_public: No such file or directory
debug1: identity file /home/n13/.ssh/id_ed25519 type -1
debug1: key_load_public: No such file or directory
debug1: identity file /home/n13/.ssh/id_ed25519-cert type -1
debug1: Enabling compatibility mode for protocol 2.0
debug1: Local version string SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.2
debug1: Remote protocol version 2.0, remote software version OpenSSH_7.2p2 Ubuntu-4ubuntu2.2
debug1: match: OpenSSH_7.2p2 Ubuntu-4ubuntu2.2 pat OpenSSH* compat 0x04000000
debug1: Authenticating to 13-yonbal.local:22 as 'n13'
debug1: SSH2_MSG_KEXINIT sent
debug1: SSH2_MSG_KEXINIT received
debug1: kex: algorithm: curve25519-sha256@libssh.org
debug1: kex: host key algorithm: ecdsa-sha2-nistp256
debug1: kex: server->client cipher: chacha20-poly1305@openssh.com MAC: <implicit> compression: none
debug1: kex: client->server cipher: chacha20-poly1305@openssh.com MAC: <implicit> compression: none
debug1: expecting SSH2_MSG_KEX_ECDH_REPLY
debug1: Server host key: ecdsa-sha2-nistp256 SHA256:ujiNJ9Jabqm7CS2pLCJW77wbclUo7g7waw3bgVdq1Dg
debug1: Host '13-yonbal.local' is known and matches the ECDSA host key.
debug1: Found key in /home/n13/.ssh/known_hosts:20
debug1: rekey after 134217728 blocks
debug1: SSH2_MSG_NEWKEYS sent
debug1: expecting SSH2_MSG_NEWKEYS
debug1: rekey after 134217728 blocks
debug1: SSH2_MSG_NEWKEYS received
debug1: SSH2_MSG_EXT_INFO received
debug1: kex_input_ext_info: server-sig-algs=<rsa-sha2-256,rsa-sha2-512>
debug1: SSH2_MSG_SERVICE_ACCEPT received
debug1: Authentications that can continue: publickey,password
debug1: Next authentication method: publickey
debug1: Offering RSA public key: /home/n13/.ssh/id_rsa
debug1: Authentications that can continue: publickey,password
debug1: Trying private key: /home/n13/.ssh/id_dsa
debug1: Trying private key: /home/n13/.ssh/id_ecdsa
debug1: Trying private key: /home/n13/.ssh/id_ed25519
debug1: Next authentication method: password
n13@13-yonbal.local's password:
debug1: Authentication succeeded (password).
Authenticated to 13-yonbal.local ([192.168.0.20]:22).
debug1: channel 0: new [client-session]
debug1: Requesting no-more-sessions@openssh.com
debug1: Entering interactive session.
debug1: pledge: network
debug1: client_input_global_request: rtype hostkeys-00@openssh.com want_reply 0
debug1: Sending environment.
debug1: Sending env LANG = en_US.UTF-8
debug1: Sending command: scp -v -f /home/n13/data/Dropbox/rsj_seminor07.pdf
Sink: Agent pid 5448
Agent pid 5448
n13@13-p50:~$ Sending file modes: C0664 7037524 rsj_seminor07.pdf
debug1: client_input_channel_req: channel 0 rtype exit-status reply 0
debug1: client_input_channel_req: channel 0 rtype eow@openssh.com reply 0
debug1: channel 0: free: client-session, nchannels 1
debug1: fd 0 clearing O_NONBLOCK
debug1: fd 1 clearing O_NONBLOCK
Transferred: sent 2368, received 2704 bytes, in 0.2 seconds
Bytes per second: sent 10393.4, received 11868.2
debug1: Exit status 1

n13@13-p50:~$
```

**Comment by 130s** [comment]
I'm pretty sure [this thread](https://unix.stackexchange.com/questions/18231/scp-fails-without-error) talks about exactly the same issue, but the solution to my case might not be that simple as tmy shell configs are hierarchical, instead of a single file.

**Comment by 130s** [comment]
As the commit message https://github.com/130s/hut_10sqft/pull/202/commits/bc6761a1801a5fa591d0f53ac945eaa6d88796ba explains, this issue is not yet resolved by #202.

**Comment by 130s** [comment]
#203 workarounds this issue.
